Cargo.toml

```
@@ -21,11 +21,19 @@ volatile-register = "0.2.0"
 bitfield = "0.13.2"
 embedded-hal = "0.2.4"
 
+[dependencies.serde_crate]
+package = "serde"
+version = "1"
+features = [ "derive" ]
+optional = true
+
 [features]
 cm7 = []
 cm7-r0p1 = ["cm7"]
 inline-asm = []
 linker-plugin-lto = []
+serde = [ "serde_crate" ]
+std-map = []
```

```suggestion
[dependencies.serde]
version = "1"
features = [ "derive" ]
optional = true

[features]
cm7 = []
cm7-r0p1 = ["cm7"]
inline-asm = []
linker-plugin-lto = []
std-map = []
```

Can't we just do this ? And get rid of the `serde(crate = "serde_crate")` on the source file ?

---

I was not aware that `namespaced-features` was stabilized. Fixed.